src/python/Utils/PortForward.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+"""
+_PortForward_
+
+A decorator for swapping ports in an url
+"""
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+
+from builtins import str
+
+import logging
+from urllib.parse import urlparse, ParseResult
+
+
+def portForward(port):
+    """
+    Decorator wrapper function for port forwarding of the REST calls of any
+    function to a given port.
+
+    Currently there are three constraints for applying this decorator.
+    1. The function to be decorated must be defined within a class and not being a static method.
+       The reason for that is because we need to be sure the function's signature will
+       always include the class instance as its first argument.
+    2. The url argument must be present as the second one in the positional argument list
+       of the decorated function (right after the class instance argument).
+    3. The url must follow the syntax specifications in RFC 1808:
+       https://tools.ietf.org/html/rfc1808.html
+
+    If all of the above constraints are fulfilled and the url is part of the
+    urlMangleList, then the url is parsed and the port is substituted with the
+    one provided as an argument to the decorator's wrapper function.
+
+    param port: The port to which the REST call should be forwarded.
+    """
+    def portForwardDecorator(callFunc):
+        """
+        The actual decorator
+        """
+        urlMangleList = ['https://tivanov',
+                         'https://alancc',
+                         'https://cmsweb']
+
+        def portManlge(callObj, url, *args, **kwargs):
```

Comment:
Todor, given that this typo is in the function name, can you please get if fixed?

Reply:
ohh definetely 
thanks for catching this one

Reply:
done